### PR TITLE
Add MVP of Regex Patterns

### PR DESCRIPTION
**Pull request by leportella**
This is a minimum implementation.

### CODE REVIEW

File: bottery/handlers.py [hunk 1]
```diff
@@ -18,6 +21,14 @@ def match(self, message):
         raise Exception('Method Not Implemented')
 
 
+class RegexHandler(BaseHandler):
+    def match(self, message):
+        compile = re.compile(self.pattern)
```
Comment (rougeth) [inline bottery/handlers.py]: `compile` is a reserved word :/

Comment (rougeth) [reply bottery/handlers.py]: Would be better if we save the compiled regex to use it later. Something like:

```
if not self.compiled_re:
    self.compiled_re = re.compile(self.pattern)

if compile.match(self.message.text):return True
        return False
```

Comment (leportella) [reply bottery/handlers.py]: ok!

**Comment by rougeth** [comment]
Thank you very much!

**Pull request merged**